src/mongo/commands.ts

```diff
@@ -30,7 +31,15 @@ export class MongoCommands {
 				throw new Error('Please connect to the database first');
 			}
 			const result = await database.executeCommand(command);
-			await util.showNewFile(result, extensionPath, 'result', '.json', activeEditor.viewColumn + 1);
+			if (command.name === 'find' || command.name === 'findOne') {
+				const db = await database.getDb();
+				const dummy = new MongoDummyNode(db.collection(command.collection), command.name);
+				dummy.data = JSON.parse(result);
+				await editor.showDocument(dummy);
+			}
+			else {
+				await util.showNewFile(result, extensionPath, 'result-mongo-ouput', '.json', activeEditor.viewColumn + 1);
```

Comment:
I felt a different name would indicate that it is not persistable. Also helps package.json correctly decide to (not) allow the update command

Reply:
I would leave the name as it was. This feels too long